plugins/modules/mysql_query.py

```diff
@@ -43,7 +43,8 @@
     - Where passed queries run in a single transaction (C(yes)) or commit them one-by-one (C(no)).
     type: bool
     default: no
-seealso:
+
+see also:
```

Comment:
```suggestion
see also:
```
this is not a typo and this key is used by parsers to generate the documentation, so please revert the change:)

Reply:
I was asking myself about that one